app/helpers/a11y_helper.rb

```diff
@@ -1,11 +1,15 @@
 # Accessibilty aka a11y helpers
 module A11yHelper
-  def icon(type, label = nil)
-    content_tag :i, '', class: "fa fa-#{type}", 'aria-label': (label || type)
+  def icon(type, label = nil, style: :normal, fixed_width: false)
+    base_class = font_awesome_class(style, fixed_width)
+
+    content_tag :i, '', class: "#{base_class} fa-#{type}", 'aria-label': (label || type)
   end
 
-  def graphical_icon(type)
-    content_tag :i, '', class: "fa fa-#{type}", 'aria-hidden': true
+  def graphical_icon(type, style: :normal, fixed_width: false)
```

Comment:
I think that `style` shouldn't have a default and we should have to be explicit each time.